src/firefly/java/edu/caltech/ipac/visualize/plot/WorldPt.java

```diff
@@ -56,16 +55,16 @@ public String serialize() {
     protected static WorldPt stringAryToWorldPt(String wpParts[]) {
         WorldPt retval= null;
         if (wpParts.length>=3) {
-            double lon= StringUtils.parseDouble(wpParts[0]);
-            double lat= StringUtils.parseDouble(wpParts[1]);
+            double lon= parseDouble(wpParts[0]);
+            double lat= parseDouble(wpParts[1]);
```

Comment:
oh, ok! great then!

On 5/16/16 6:57 PM, Trey Roby wrote:
>
> In src/firefly/java/edu/caltech/ipac/visualize/plot/WorldPt.java 
> <https://github.com/Caltech-IPAC/firefly/pull/86#discussion_r63455053>:
>
> > @@ -56,16 +55,16 @@ public String serialize() {
> >      protected static WorldPt stringAryToWorldPt(String wpParts[]) {
> >          WorldPt retval= null;
> >          if (wpParts.length>=3) {
> > -            double lon= StringUtils.parseDouble(wpParts[0]);
> > -            double lat= StringUtils.parseDouble(wpParts[1]);
> > +            double lon= parseDouble(wpParts[0]);
> > +            double lat= parseDouble(wpParts[1]);
>
> It is doing the same as before. I move the parse double to the base 
> class Pt.java. I am trying to remove the dependency on StringUtils to 
> make the new GWT cross compile smaller.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Caltech-IPAC/firefly/pull/86/files/3dddf13ead9a8b65a93254dce2ab5fcd6f9f4283#r63455053>
>

-- 
Emmanuel Joliet
NASA/IPAC  Infrared Science Archive
MS 100-22, Caltech, Pasadena, CA 91125
Office: 170
Email:  ejoliet@ipac.caltech.edu
Phone:  (626) 395-1489